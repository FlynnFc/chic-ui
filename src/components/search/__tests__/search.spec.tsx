import Search from '..';
import { render } from '../../../test/utils';

/* eslint-disable */
describe('Search Component', () => {
  it('Renders Correctly', () => {
    const SearchGroup = render(
      <>
        <Search
          placeholder="Search something..."
          width="400px"
          value="Value"
          onChange={(e) => alert(`Value Changed`)}
        />
        ;
      </>
    );
    expect(SearchGroup).toMatchSnapshot();
  });
});
