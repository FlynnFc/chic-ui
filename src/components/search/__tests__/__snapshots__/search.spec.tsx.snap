// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Component Renders Correctly 1`] = `
Array [
  <div
    className="sc-iBPTik bbcsVj"
    width="400px"
  >
    <div
      className="sc-hKgJUU ibQTEz"
      width="100%"
    >
      <input
        className="sc-eCstlR eeQFSX"
        disabled={false}
        onChange={[Function]}
        placeholder="Search something..."
        readOnly={false}
        type="text"
        value="Value"
        width="100%"
      />
      <div
        className="sc-jSgvzq kIBQXf"
      />
      <svg
        className="sc-gKseQn ewXZdy"
        innerSize="default"
        onClick={[Function]}
        viewBox="0 0 490 490"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M456.851 0L245 212.564 33.149 0 .708 32.337l211.961 212.667L.708 457.678 33.149 490 245 277.443 456.851 490l32.441-32.322-211.961-212.674L489.292 32.337z"
        />
      </svg>
    </div>
    <button
      className="sc-bdfBQB iGjbod"
      disabled={false}
      size="default"
      type="button"
    >
      Search
    </button>
  </div>,
  ";",
]
`;
